src/components/Title.js

```diff
@@ -1,7 +1,12 @@
 import React from "react";
 
+// this is a component
 const Title = () => {
```

Comment:
the title component is supposed to take in props as mentioned in your issue. We will not hard code "What is WINC?" in this file.